internal/pkg/cli/deploy/deploy.go

```diff
@@ -61,9 +62,13 @@ const (
 )
 
 var (
-	aliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
+	rdwsAliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
 		color.HighlightCode("http.alias"),
 		color.HighlightCode("copilot app init --domain example.com"))
+	ecsAliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s, or import any certificates to your environment: %s.\n",
```

Comment:
Should we separate with new lines since the text is long?
```suggestion
	ecsAliasUsedWithoutDomainFriendlyText = fmt.Sprintf(`To use %s, your application must be:
* Associated with a domain: %s
* Or, using imported certificates to your environment: %s
`,
```